index.html

```diff
@@ -14,21 +14,28 @@
     Hello Camper!
 
     For now, the test suite only works in Chrome!
+
     Please read the README below in the JS Editor before beginning.
+
     Feel free to delete this message once you have read it.
     Good luck and Happy Coding!
 
+
     - The freeCodeCamp Team
 
+
     -->
```

Comment:
kindly remove unmeaningful comments to your code.  The comments need to be relevant to the code